data/json/mapgen/dairy_farm.json

```diff
@@ -0,0 +1,244 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "dairy_farm_NW" ],
+    "weight": 250,
+    "object": {
+      "rows": [
+        "........................",
+        ".$$$$$$$$$$$$$$$$$$$$$$$",
+        ".$......................",
+        ".$......................",
+        ".$.................... .",
+        ".$......................",
+        ".$......................",
+        ".$......................",
+        ".$......................",
+        ".$......................",
+        ".$......................",
+        ".$......................",
+        ".$......................",
+        ".$......................",
+        ".$......................",
+        ".$......................",
+        ".$......................",
+        ".$................... ..",
+        ".$................... ..",
+        ".$................... ..",
+        ".$..................... ",
+        ".$......................",
+        ".$......................",
+        ".$......................"
+      ],
+      "terrain": {
+        " ": "t_grass",
+        "$": "t_chainfence_v",
+        ".": "t_grass"
```

Comment:
Grass should be trampled/eaten. You can add chances for dirt like this:
```
".": ["t_grass", "t_grass", "t_dirt", "t_dirt", "t_dirt"]
```
The above would give a 60% chance for dirt, 40% chance for grass.